tests/domain/usecases/fitbit/test_usecase_process_daily_activities.py

```diff
@@ -43,7 +43,7 @@ class DailyActivityScenario:
     • Activity count: 2
     • Total duration: 15 minutes ↗️ New record (last 180 days)! 🏆
     • Total calories: 250 ↗️ New record (last 180 days)! 🏆
-    • Distance: 15.000 km ⬆️ New record (last 180 days)! 🏆
+    • Distance: 15.000 km ⬆️ New record (last 180 days)! 🏆 1 day streak! 👏
```

Comment:
Add new test cases:
* We didn't reach our goal today
* There's no activity at all for today for the given type (this test is interesting even without the new feature).